Package.swift

```diff
@@ -11,15 +11,34 @@ let package = Package(
     ],
     targets: [
         .target(name: "COpenCombineHelpers"),
-        .target(name: "OpenCombine", dependencies: ["COpenCombineHelpers"]),
+        .target(
+          name: "OpenCombine", 
+          dependencies: [
+            .target(name: "COpenCombineHelpers", condition: .when(platforms: [.macOS, .iOS, .watchOS, .tvOS, .linux]))
+          ],
+          exclude: [
+            "Publishers/Publishers.Encode.swift.gyb",
+            "Publishers/Publishers.MapKeyPath.swift.gyb",
+            "Publishers/Publishers.Catch.swift.gyb"
+          ]
```

Comment:
Could you please explain why is this necessary?

Reply:
Swift 5.3 package manifests require you either to add all files in the `Sources` as SwiftPM resources, or to exclude them explicitly.